### PR TITLE
Check for MSBuild version is higher than required by the .NET SDK

### PR DESCRIPTION
**Pull request by JoeRobich**
Provide a warning when the discovered MSBuild version is lower than the minimumMSBuildVersion supported by the configured SDK.

### CODE REVIEW

File: src/OmniSharp.Host/MSBuild/Discovery/Extensions.cs [hunk 1]
```diff
@@ -55,12 +87,12 @@ public static bool HasDotNetSdksResolvers(this MSBuildInstance instance)
         /// Checks if it is MSBuild from Visual Studio 2017 RTM that cannot be used.
```
Comment (filipw) [inline src/OmniSharp.Host/MSBuild/Discovery/Extensions.cs:87]: this comment is long time incorrect 😁

File: src/OmniSharp.Host/MSBuild/Discovery/Extensions.cs [hunk 2]
```diff
@@ -112,5 +144,28 @@ private static int GetInstanceFeatureScore(MSBuildInstance i)
...
+            if (dotNetInfo is null
+                || string.IsNullOrWhiteSpace(dotNetInfo.SdksPath)
+                || dotNetInfo.SdkVersion is null)
+            {
+                return s_minimumMSBuildVersion;
+            }
+
+            var version = dotNetInfo.SdkVersion;
+            var sdksPath = dotNetInfo.SdksPath;
+            var minimumVersionPath = Path.Combine(sdksPath, version.ToNormalizedString(), "minimumMSBuildVersion");
```
Comment (filipw) [inline src/OmniSharp.Host/MSBuild/Discovery/Extensions.cs:159]: this is really there?! that's new to me!

File: src/OmniSharp.Host/MSBuild/Discovery/Extensions.cs (hunk 1 above)
Comment (JoeRobich) [reply src/OmniSharp.Host/MSBuild/Discovery/Extensions.cs]: Updated

File: src/OmniSharp.Host/MSBuild/Discovery/Extensions.cs (hunk 2 above)
Comment (JoeRobich) [reply src/OmniSharp.Host/MSBuild/Discovery/Extensions.cs:166]: I was happy to find it =)

**Pull request merged**